api_core/setup.py

```diff
@@ -12,68 +12,72 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import io
 import os
 
-from setuptools import find_packages
-from setuptools import setup
+import setuptools
 
 
-PACKAGE_ROOT = os.path.abspath(os.path.dirname(__file__))
+# Package metadata.
+
+name = 'google-api-core'
+description = 'Google API client core library'
+version = '0.1.5.dev1'
+# Should be one of:
+# 'Development Status :: 3 - Alpha'
+# 'Development Status :: 4 - Beta'
+# 'Development Status :: 5 - Stable'
+release_status = 'Development Status :: 4 - Beta'
+dependencies = [
+    'googleapis-common-protos<2.0dev,>=1.5.3',
+    'protobuf>=3.0.0',
+    'google-auth<2.0.0dev,>=0.4.0',
+    'requests<3.0.0dev,>=2.18.0',
+    'setuptools>=34.0.0',
+    'six>=1.10.0',
+    'pytz',
+]
+extras = {
+    'grpc': 'grpcio>=1.8.2',
+    ':python_version < "3.2"': 'futures>=3.2.0',
+}
 
-with open(os.path.join(PACKAGE_ROOT, 'README.rst')) as file_obj:
-    README = file_obj.read()
 
+# Setup boilerplate below this line.
 
-SETUP_BASE = {
-    'author': 'Google Cloud Platform',
-    'author_email': 'googleapis-publisher@google.com',
-    'scripts': [],
-    'url': 'https://github.com/GoogleCloudPlatform/google-cloud-python',
-    'license': 'Apache 2.0',
-    'platforms': 'Posix; MacOS X; Windows',
-    'include_package_data': True,
-    'zip_safe': False,
-    'classifiers': [
-        'Development Status :: 4 - Beta',
+package_root = os.path.abspath(os.path.dirname(__file__))
+
+with io.open(os.path.join(package_root, 'README.rst')) as readme_file:
```

Comment:
Do you want to explicitly pass in the encoding in case we ever decide to add some unicode to a README?